.github/workflows/publish-stable.yml

```
@@ -12,6 +12,28 @@ on:
       - completed
 
 jobs:
+  generate-metadata-for-extension:
+    # Generate metadata for extension before automerging the current branch to master
+    # only run if prechecks pass and the source branch is named 'prerelease/*'
+    if: ${{ startsWith(github.head_ref, 'prerelease/1') }} && ${{ github.event.label.name == 'automerge' }}
+    runs-on: ubuntu-latest
+    steps:
+    - name: generate meta for extension
+      uses: actions/checkout@v2
+    - name: install dependencies
+      run: yarn
+    - name: yarn meta:extension
+      run: yarn meta:extension
+    - name: check for changes
+      run: git status
+    - name: stage changed files
+      run: git add .
+    - name: commit changed files
+      run: git commit -m "Generated extension data"
+    - name: push code
+      run: git push
```

nit: easier to put this altogether imo

```suggestion
- name: commit changes
   run: |
     git add .
     git commit -m "Add extension data"
     git push
```

---

done